packages/vscode-extension/package.nls.cs.json

```diff
@@ -351,16 +351,16 @@
   "teamstoolkit.survey.remindMeLater.title": "Připomenout později",
   "teamstoolkit.survey.takeSurvey.message": "takeSurvey",
   "teamstoolkit.survey.takeSurvey.title": "Vyplnit dotazník",
-  "teamstoolkit.tutorials.addSso.detail": "Vývoj funkce jednotného přihlášení pro spouštění stránek Teams a funkcí robota",
-  "teamstoolkit.tutorials.addSso.label": "Přidat jednotné přihlašování",
-  "teamstoolkit.tutorials.commandAndResponse.detail": "Automatizace opakujících se úloh pomocí jednoduchého příkazu a odpovědi v konverzaci",
-  "teamstoolkit.tutorials.commandAndResponse.label": "Sestavení příkazu a odpovědi",
-  "teamstoolkit.tutorials.connectApi.detail": "Připojit k rozhraní API s podporou ověřování pomocí sady TeamsFx SDK",
-  "teamstoolkit.tutorials.connectApi.label": "Připojit se k rozhraní API",
-  "teamstoolkit.tutorials.embedWebPages.detail": "Přineste si vlastní statické webové stránky a vložte je do Microsoft Teams",
-  "teamstoolkit.tutorials.embedWebPages.label": "Vložit existující webové stránky",
-  "teamstoolkit.tutorials.sendNotification.detail": "Odesílání oznámení do Teams z webových služeb pomocí robota nebo příchozího webhooku",
-  "teamstoolkit.tutorials.sendNotification.label": "Odeslat oznámení do Teams",
+  "teamstoolkit.guides.addSso.detail": "Vývoj funkce jednotného přihlášení pro spouštění stránek Teams a funkcí robota",

```

Comment:
No need to change other languages, they'll be overwriten

Reply:
Thanks, updated.